### DISCUSSION TITLE
Can't download from https://hotaudio.net/

### DISCUSSION THREAD

**Discussion by Merlin-007** [aclap-dev/video-downloadhelper#965] [Help and Q&A]
### Your question

Can't download from https://hotaudio.net/ 

### Addon & CoApp version and details

Nothing is detected.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

This is not a problem report that will get you any help.  You need to actually follow the instructions that you ignored when you created this problem.  You need to provide the proper information.  Click the Wiki link at the top of any page of this forum.  That takes you to what is basically a user manual.  Down the right side of that page there is a table of contents.  But it's not fully expanded.  At the bottom, there is a clickable button that says `Show 8 more pages...` that you must click.  That will reveal a chapter with the title `Where can I find my Versions?`  You need to click that, read what you find, & follow the instructions.  You need to include that information in the section above under the heading `Addon & CoApp version and details` that you failed to fill in.   Then you need to supply specific URLs of web pages on which you are having trouble.  Don't just name that site.  Go to a web page where a video is hosted that is giving you trouble & paste the URL of that page into your problem report.  If you think it would clarify the problem, post a few such URLs.  If you want help, you should make it easy for somebody who wants to help you.

Above your post to the right you can see 3 dots.  Click the 3 dots.  That will drop down a menu.  You should be able to figure out how you can repair your faulty problem report.
